test-lib.sh

```diff
@@ -0,0 +1,142 @@
+#
+# Test a container image.
+#
+# Always use sourced from a specific container testfile 
+#
+# reguires definition of CID_FILE_DIR
+# CID_FILE_DIR=$(mktemp --suffix=<container>_test_cidfiles -d)
+# reguires definition of TEST_LIST 
+# TEST_LIST="\
+# run_container_creation_tests
+# run_doc_test <words_to_look_for_in_the_doc>"
+
+# Container CI tests
+# abbreviated as "concit"
```

Comment:
Could be OK, but @pkubatrh, @omron93, @hhorak FYI.  Ideas?

I have no strong preference, but maybe `ct_` prefix could be enough  (Container Testing).

Once we claim we provide "library", it will be much harder to migrate to different function prefix.  So we should rather agree that we use pretty namespace.